### DISCUSSION TITLE
建议把“拒绝/放弃”标签替换成“搁置/难以实现”

### DISCUSSION THREAD

**Discussion by Deep-Dark-Forest** [Meloong-Git/PCL#1823] [讨论]
拒绝感觉有点太直接了，如果是因为需要的人太少就叫“搁置”呗，以后需要也可以重开issue.......
如果因为太难做而难以实现，那就“难以实现”呗

**Comment by Chiloven945** [comment]
放弃是一个动作，难以实现是对放弃这个动作的修饰，所以说放弃并不意味着都是因为难以实现而放弃，也可以因为很多其他的原因
拒绝直接并没有什么不好的，对于并不好的事情就得说不，不可能这惯着那惯着的
综上所述，目前的标签已经是概括性较强能满足需求的了

**Comment by iPodtouch6557** [comment]
我个人认为可以改，因为看到`拒绝/放弃`心里确实会很不舒服，不是这惯着那惯着的问题
~比如我以前交了将近20个issue都没采纳后看到拒绝放弃心里就硌得慌~

**Comment by LTCatt** [comment]
一些很离谱的建议（例如说启动 MCD）我也只能说拒绝 =。=
GitHub 本身就是 **技术性很强** 的地方，而不是 **需要考虑个人情绪** 的论坛，所以不打算修改。
